web/store/issue/issue.store.ts

```diff
@@ -18,7 +18,7 @@ export type IIssueStore = {
   removeIssue(issueId: string): void;
   // helper methods
   getIssueById(issueId: string): undefined | TIssue;
-  getIssuesByIds(issueIds: string[]): undefined | Record<string, TIssue>; // Record defines issue_id as key and TIssue as value
+  getIssuesByIds(issueIds: string[], type: "archived" | "un-archived"): undefined | Record<string, TIssue>; // Record defines issue_id as key and TIssue as value
```

Comment:
@aaryan610  Make the parameter `type` optional and the default value "un-archived". So that we don't need to specify all the times wherever we are calling this function.